src/pages/RestrictedAction/Workspace/WorkspaceAdminRestrictedAction.tsx

```diff
@@ -28,7 +27,7 @@ function WorkspaceAdminRestrictedAction({policyID}: WorkspaceAdminRestrictedActi
 
     const openAdminsReport = useCallback(() => {
         const reportID = `${PolicyUtils.getPolicy(policyID)?.chatReportIDAdmins}` ?? '-1';
-        Report.openReport(reportID);
+        Navigation.resetToHome();
```

Comment:
Same reasoning. we can use Navigation.closeRHPFlow(); instead

Reply:
Adjusted